analyzers/JoeSandbox/JoeSandbox_File_Analysis_Inet.json

```diff
@@ -41,6 +41,30 @@
       "multi": false,
       "required": true,
       "defaultValue": 30
-    }
+    },
+    {
+      "name": "HTML_report",
+      "description": "Download HTML report",
+      "type": "boolean",
+      "multi": false,
+      "required": true,
+      "defaultValue": false 
+    },
+	{
+      "name": "images",
+      "description": "Allow images in the report",
+      "type": "boolean",
+      "multi": false,
+      "required": true,
+      "defaultValue": false 
+    },
+	{
+	  "name": "observables",
+      "description": "Creat observables form report",
+      "type": "boolean",
+      "multi": false,
+      "required": true,
+      "defaultValue": false 
+	}
   ]
 }
```

Comment:
The doc for Analyzers and Responders is generated automatically, based upon json files, `readme.md` file and pictures. Can you update the analyzer with the missing content, please ? 
You can look at the documentation here: [https://thehive-project.github.io/Cortex-Analyzers/dev_guides/analyzers_definition/](https://thehive-project.github.io/Cortex-Analyzers/dev_guides/analyzers_definition/).
You can also get inspired from one of existing analyzers like [https://github.com/TheHive-Project/Cortex-Analyzers/tree/master/analyzers/EmlParser](https://github.com/TheHive-Project/Cortex-Analyzers/tree/master/analyzers/EmlParser) and see the result here: [https://thehive-project.github.io/Cortex-Analyzers/analyzers/EmlParser/](https://thehive-project.github.io/Cortex-Analyzers/analyzers/EmlParser/)